samples/bluetooth/df_per_adv_beacon/README.rst

```diff
@@ -0,0 +1,65 @@
+.. _bluetooth_df_per_adv_beacon:
+
+Bluetooth: Direction Finding Periodic Advertising Beacon
+########################################################
+
+Overview
+********
+
+A simple application demonstrating the BLE Direction Finding CTE Broadcaster functionality by sending Constant Tone Extension with periodic advertising PDUs.
```

Comment:
Not 100% sure since I'm reviewing this in a browser, but doesn't this go beyond the 100 character line length limit (same for other parts of this file as well)

Reply:
It was over 100 characters in a line. I've thought that RST files are not limited to 100 chars per line. 

Line breaks added to fit into the limit.